src/win.py

```diff
@@ -0,0 +1,221 @@
+from constant.board import BOARD_SIZE, WIN_CONDITION, WIN_AREA_CHECK
+
+def is_position_outside_board(position):
+    return position[0] < 0 or position[0] >= BOARD_SIZE or position[1] < 0 or position[1] >= BOARD_SIZE
+
+def browse_board_in_direction_to_get_aligmnent(board, start_position, direction, results):
```

Comment:
in this function, you hide regarding the signature that you modify the results. It should be better to return the results and add them in your main array